proto/obb/session.proto

```diff
@@ -57,67 +55,107 @@ message InvalidSessionReason {
 enum SessionStatus {
   // Not a valid value. Used for blocking 0 as invalid state.
   SESSION_STATUS_UNSPECIFIED = 0;
+
+  // The session is valid and can still be used.
   SESSION_STATUS_VALID = 1;
+
+  // Session is not valid anymore.
   SESSION_STATUS_INVALID = 2;
 }
 
-message SessionRequest {
-  oneof data {
-    SessionCreate create = 1;
-    StreamKeepalive keepalive = 2;
-    SessionInfo info = 3;
-  }
+message SessionCreateRequest {
+  // List of desired selections to include in the new session.
+  // Format: "<event_id>/<market_id>/<outcome_id>?<market_specifier>"
+  // e.g. “od:match:1234/1/1?map=1&way=two”
+  repeated string selection_ids = 2;
 }
 
-message SessionResponse {
-  oneof data {
-    SessionCreateState create = 1;
-    StreamKeepalive keepalive = 2;
-    SessionInfoState info = 3;
+message SessionCreateResponse {
+  message CreatedSession {
+    // List of selections used in the current session.
+    repeated SessionSelection selections = 1;
+
+    // Available markets for the new session with the current session selections.
+    repeated SessionMarket markets = 2;
+
+    // Final odds for the selected combination of picks.
+    // Odds multiplied by 10000 and rounded to uint value.
+    uint64 odds = 3;
```

Comment:
Nitpicking (but it is public API): I would move `odds=3` property above `markets=2` property  to unify properties ordering as it is in `SessionInfoRequest`

Reply:
reordered